lib/components/viewers/pattern-viewer.tsx

```diff
@@ -136,7 +153,7 @@ const PatternViewer = ({
     )
   }
 
-  return null
+  return <p>oops we cannot find this pattern.</p>
```

Comment:
Nit - intl

Reply:
🫣 Left this in by mistake haha thanks for catching! If we redirect to a real pattern, there shouldn't be any reason for this